src/mutations/events/events.ts

```diff
@@ -31,11 +31,12 @@ const createEvent = async (
     image_url: "",
     organiser: {
       connect: {
-        id: user.id,
+        username: user.username,
       },
```

Comment:
umm why this change here? shouldn't `id` always exist on `user`?

Reply:
Oh this was definitely an experimental change. Going to revert this.